### PR TITLE
net-misc/aggregate-flim: update HOMEPAGE

### PR DESCRIPTION
**Pull request by wimmuskee**
Closes: https://bugs.gentoo.org/680392
Signed-off-by: Wim Muskee <wimmuskee@gmail.com>

### CODE REVIEW

File: net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild [hunk 1]
```diff
@@ -1,19 +1,17 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=0
+EAPI=7
```
Comment (juippis) [inline net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild]: This requires a revbump, and a revbump requires resetting the keywords (from stable to ~). Also if you decide to update and revbump, the `dodoc ...` can be replaced with a simple `einstalldocs`.

File: net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild [hunk 2]
```diff
@@ -1,19 +1,17 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Foundation
```
Comment (juippis) [inline net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild]: If you commit with repoman, it fixes this header for you automatically. You still use old "Foundation".

File: net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild (hunk 1 above)
Comment (wimmuskee) [reply net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild]: Set back to EAPI 0.

File: net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild (hunk 2 above)
Comment (wimmuskee) [reply net-misc/aggregate-flim/aggregate-flim-1.0.2.ebuild]: Fixed, and fixed also in my other open pull requests.

**Comment by robbat2** [comment]
Merged, thanks.

**Pull request closed without merging**